node/package.json

```diff
@@ -35,7 +36,9 @@
         "fix-protobuf-file": "replace 'this\\.encode\\(message, writer\\)\\.ldelim' 'this.encode(message, writer && writer.len ? writer.fork() : writer).ldelim' src/ProtobufMessage.js",
         "test": "npm run build-test-utils && jest --verbose --runInBand --testPathIgnorePatterns='RedisModules'",
         "build-test-utils": "cd ../utils && npm i && npm run build",
-        "lint": "eslint -f unix \"src/**/*.{ts,tsx}\"",
+        "lint:fix": "npm run install-linting && npx eslint -c ../eslint.config.mjs --fix",
+        "lint": "npm run install-linting && npx eslint -c ../eslint.config.mjs",
```

Comment:
These should include prettier as well

Reply:
Thought about it, didn't know if do or not. If you think so its enough for me, will do.